sagemaker_processing/fairness_and_explainability/fairness_and_explainability.ipynb

```diff
@@ -75,6 +75,10 @@
     "region = session.boto_region_name\n",
     "# Define IAM role\n",
     "from sagemaker import get_execution_role\n",
+    "from sagemaker.serializers import CSVSerializer\n",
+    "from sagemaker.deserializers import CSVDeserializer\n",
+    "import matplotlib\n",
+    "import sklearn\n",
```

Comment:
Do we want to use `scikit-learn` instead of `sklearn` since it's deprecated?

Related: https://pypi.org/project/sklearn/ and https://t.corp.amazon.com/V781770294

Reply:
Nvm, please ignore my comment. Saw it in the requirements.txt